library/tiqr/Tiqr/Message/GCM.php

```diff
@@ -0,0 +1,108 @@
+<?php
+/**
+ * This file is part of the tiqr project.
+ * 
+ * The tiqr project aims to provide an open implementation for 
+ * authentication using mobile devices. It was initiated by 
+ * SURFnet and developed by Egeniq.
+ *
+ * More information: http://www.tiqr.org
+ *
+ * @author Peter Verhage <peter@egeniq.com>
+ * 
+ * @package tiqr
+ *
+ * @license New BSD License - See LICENSE file for details.
+ *
+ * @copyright (C) 2010-2011 SURFnet BV
```

Comment:
For new file maybe update to 2015. (Or remove from the header since most year indicators here will be outdated)